packages/gestalt/src/Sheet.js

```diff
@@ -142,6 +150,11 @@ function Sheet(props: SheetProps): Node {
     };
   }, [onDismiss]);
 
+  const handleOnAnimationEnd = useCallback(() => {
+    onAnimationEndFromHook?.();
+    onAnimationEnd?.({ animationState: animationStateHook === 'in' ? 'in' : 'out' });
```

Comment:
Can this not be simplified to "animationState: animationStateHook"?

Reply:
@ayeshakmaz I didn't understand the question, can you rephrase it?

Reply:
OOH! `animationStateHook` returns only 'in' & 'out' but the type is `export type AnimationStateType = 'in' | 'postIn' | 'out' | 'postOut' | null;` We wanna simplify the type to just   `'in' | 'out'` and we just pass `animationStateHook` we'd have to extend its type. We can only access 'in' & 'out' at the end of the animation, but the type doesn't match with the behavior in this particular moment.